applications/app/views/survey/emailDigest404.scala.html

```diff
@@ -0,0 +1,45 @@
+@(surveyPage: Page)(implicit request: RequestHeader)
+
+@import model.Page
+@import conf.Configuration.Survey.formStackAccountName
```

Comment:
The idea was to pass an arbitrary string from the endpoint to the controller to the view, meaning we could view any form. Could do this, but then it requires a release for new forms? At least then we could see what forms are available.